maven/bnd-maven-plugin/README.md

```diff
@@ -180,6 +180,28 @@ The plugin will by default set some OSGi bundle headers derived from [pom elemen
 | `Bundle-Developers`   | `developers` (child element `id` must be set on each developer) |
 | `Bundle-DocURL`       | `url`                                                           |
 
+### POM Properties
+
+The Bnd instructions can reference properties defined for the Maven project and Maven settings using the `project.` and `settings.` stems, respectively.
+Other properties defined in the POM can also be referenced.
+
+```xml
+<properties>
+	<implementation.vendor>Acme Inc.</implementation.vendor>
+</properties>
+<plugin>
+  <groupId>biz.aQute.bnd</groupId>
+  <artifactId>bnd-maven-plugin</artifactId>
+  <configuration>
+    <bnd><![CDATA[
```

Comment:
@bjhargrave maybe this is not the best example, as with inline bnd instructions the replacement is actually done by Maven. Only for external bnd files it is done by the bnd-maven-plugin IIUC.

Reply:
Perhaps you can make a PR to improve the example?

Reply:
Either split up the example to show both `pom.xml` and `bnd.bnd` or get rid of it alltogether I would say. But referencing https://maven.apache.org/ref/current/maven-model-builder/#model-interpolation in this context would be beneficial, as it works similarly.